build/98.css

```diff
@@ -102,6 +102,7 @@ button:not(:disabled):active {
   box-shadow: inset -1px -1px #ffffff,
     inset 1px 1px #0a0a0a, inset -2px -2px #dfdfdf,
     inset 2px 2px #808080;
+  padding: 2px 11px 0 13px;
```

Comment:
I had the exact same fix. Just to note for posterity on some of the idiosyncrasies of the fix:
* `2px` was required due to Chrome not moving a `1px` value. There appears to be a conflicting rule in css resolution there
* `0px` for the bottom as negative values aren't valid in FireFox and Chrome for `padding`. Interestingly, full `padding-top`, `right`, etc. does actually work. Issue is specifically with the condensed `padding` tag for both browsers.